### ISSUE TITLE
move add new holiday button

### ISSUE THREAD

**Issue by illori** [SimpleMachines/SMF#2946]
maybe a good idea? move it to the top of the list instead of the bottom. might need a little tweaking but i dont think it hurts to move it. code below to move it.

![](http://imagizer.imageshack.us/v2/825x114q90/909/9qCOml.png)

ManageCalendar.php

```
    'additional_rows' => array(
            array(
                'position' => 'above_column_headers',
                'value' => '<a class="button_link" href="' . $scripturl . '?action=admin;area=managecalendar;sa=editholiday">' . $txt['holidays_add'] . '</a>',
                'style' => 'text-align: right;',
            ),      
            array(
                'position' => 'below_table_data',
                'value' => '<input type="submit" name="delete" value="' . $txt['quickmod_delete_selected'] . '" class="button_submit">',
            ),
        ),
```

**Comment by margarett** [comment]
It's not like the idea isn't good. But it will break the usual aspect of SMF's lists which usually have the "Add" button at the bottom of the page.

And Calendar? Does anyone really use it yet? :tongue:

**Comment by illori** [comment]
maybe worth moving it for the others as well.

calendar was an example that was requested on sm.org,

**Comment by Kindred-999** [comment]
Hmmm  dunno -- I actually prefer it at the bottom myself (for all lists)

**Comment by Oldiesmann** [comment]
Why not have it in both places?

**Comment by illori** [comment]
Sounds good to me.

**Comment by margarett** [comment]
Well... If want to do it in all places, yes. If not I think it will break the "standard" layout of lists as I said before...

**Comment by illori** [comment]
This would also restore the buttons on the error log page to the top where
they used to be.

**Comment by Oldiesmann** [comment]
#2960 addresses this and also makes similar changes to several other lists in the admin center (but not all of them - it would look weird having buttons at the top of the "Attachment directory paths" list for instance). Note that it will look a bit weird due to the "Remove Selected" button being taller than the "Add Holiday" one, but we can tweak CSS to address that as needed.

**Comment by Oldiesmann** [comment]
Going to close this since the main issue here was fixed with #2960.
